robottelo/ui/contenthost.py

```diff
@@ -170,9 +170,9 @@ def package_search(self, name, package_name):
         """Search for installed package on specific content host"""
         self.click(self.search(name))
         self.click(tab_locators['contenthost.tab_packages'])
-        self.assign_value(
-            locators['contenthost.package_search_box'], package_name)
-        self.click(locators['contenthost.package_search_button'])
+        self.click(tab_locators.contenthost.tab_packages.installed)
```

Comment:
maybe, we like to have option to provide what tab we like to click?
`def package_search(self, name, package_name, tab_name='Installable'):`
Just a suggestion

Reply:
Docstring says: "Search for installed package". So I just implemented exactly this behavior :)
But I guess you're right.